edx_proctoring/api.py

```diff
@@ -2085,6 +2085,20 @@ def is_backend_dashboard_available(course_id):
     return False
 
 
+def does_course_support_onboarding(course_id):
+    """
+    Returns whether this course supports onboarding exams.
+    """
+    exams = ProctoredExam.get_all_exams_for_course(
+        course_id,
+        active_only=True
+    )
+    for exam in exams:
+        if get_backend_provider(name=exam.backend).supports_onboarding:
+            return True
+    return False
```

Comment:
I'd probably add a comment to the docstring that this can be optimized to not iterate, but rather to check only one exam. We don't need to do that now, but we should retain the context that we don't support multiple backends per course and as such can optimize if necessary.